Release_Process.md

```diff
@@ -59,3 +59,12 @@ These steps are common between minor and patch releases:
         (without the 'v' in front) is _not_ active.
      3. Make sure the **stable** branch is _not_ active.
      4. Scroll to the bottom of the page and click the Submit button.
+1. Update all deployments with the currently released version accordingly.
```

Comment:
Also, I prefer 2 separate commits as the release might succeed but deployments might need to be changed. For example, open new ports, modify NGINX accordingly, update k8s yaml files, enable/disable security.
We have separate concerns targeted at each commit that way.